src/modules/common/components/div/Div.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
```

Comment:
same here with file naming `Div.js` the source code might get polluted with different naming patterns, please change it to current format and if you have suggestions please propose them as improvements 🙏🏼

Reply:
You mean folder (should be uppercase) and not the component, right? 

Btw, which is the correct pattern? I see very different patterns used, like in the files i work in now. 
<img width="272" alt="image" src="https://user-images.githubusercontent.com/8784876/206668146-7c172fde-1a5a-4ee3-9e90-2867ce2827b0.png">

Sometimes we expose a "private" component (uppercase) with index, sometimes folders only have index (lowercase) and sometimes components in folders are lowercase. The consistent thing are that component folders start with uppercase.

Or am i missing some pattern here 🤔

Reply:
😅 thats a fair point, we do not have a standard at the moment, everyone using their own style i think.

Here is my inputs:
- camelCase: hooks
- Pascal: Components, Directory

Probably we should first define and address it for the whole project.

Reply:
Yes, this would be great to define. I think it has become mixed up when introducing the index pattern, or reversed (moving away from index pattern).

Some notes

**What everybody probably will agree on is that React Components should be Pascal**  
And my input would be that almost everything else should be camelCase. (And to me, the .css module that is part of the component should also be Pascal, while global css should be camelCase or hyphen).

**The index pattern** 
`/IndexComponent/index.js` can be imported like this:
`import IndexComponent from '@x/IndexComponent';`
To make the directory Pascal here make sense, With an index file the directory "is" _the_ component because of the way it is imported. 

**The none index pattern** 
`/indexComponent/IndexComponent.js` can be imported like this:
`import IndexComponent from '@x/indexComponent/IndexComponent';`
To make the directory Pascal here does not make as much sense in my opinion because it can no longer be seen as the component.

**Other files**
hyphen-case: some-example-images.png, 1-circleci.yml